packages/components/src/components/modal/modal.jsx

```diff
@@ -17,13 +17,13 @@ class ModalElement extends React.PureComponent {
     }
 
     componentDidMount = () => {
-        document.addEventListener('mousedown', this.handleClickOutside);
+        if (this.props.is_closed_on_outside_click) document.addEventListener('mousedown', this.handleClickOutside);
         this.el.classList.add('dc-modal');
         this.state.modal_root.appendChild(this.el);
     };
 
     componentWillUnmount = () => {
-        document.removeEventListener('mousedown', this.handleClickOutside);
+        if (this.props.is_closed_on_outside_click) document.removeEventListener('mousedown', this.handleClickOutside);
```

Comment:
```suggestion
        if (this.props.is_closed_on_outside_click) { 
            document.removeEventListener('mousedown', this.handleClickOutside);
        }
```

Reply:
@zaki-binary i thought mobile was suppose to close always on click outside? why you have to add it here?